elasticsearch_metrics/__init__.py

```diff
@@ -3,6 +3,8 @@
 from elasticsearch_dsl.connections import connections
 from django.utils.module_loading import autodiscover_modules
 
+from .field import *  # noqa
```

Comment:
On second thought, tt might be better to do this in the `metrics` model rather than at the top level, for consistency with `django.db.models`.

```python
from elasticsearch_metrics import metrics

class MyMetric(metrics.Metric):
    my_keyword = metrics.Keyword()
```

Reply:
I've made this change.